views/reviews/index.ejs

```diff
@@ -54,8 +54,8 @@
                         <a class="btn btn-primary <% if (currentUser && campground.reviews.some(function (review) {return review.author.id.equals(currentUser._id)})) { %> disabled <% } %>" href="/campgrounds/<%= campground._id %>/reviews/new">
                             Write a New Review</a>
                     </div>
-                    <div style="margin-top: 10px;">
-                        <a class="btn btn-default" href="/campgrounds/<%= campground._id %>"><i class="fa fa-arrow-left"></i> Go back</a>
+                    <div class="reviews-back-button">
```

Comment:
It's actually a button container, not a button itself, so it could be a good idea to update a class name based on what it is.